### ISSUE TITLE
What about file uploads?

### ISSUE THREAD

**Issue by titusz** [ephes/django_fileresponse#1]
A similar optimization could be done for file uploads. Django ASGI handlers reads the request body with an async function, but it seems write the incoming data with a blocking write call to `SpooledTemporaryFile`:  https://github.com/django/django/blob/main/django/core/handlers/asgi.py#L191

Just an idea. Would love to hear what you think about it.

**Comment by ephes** [comment]
Yes, it would be possible to do this for file uploads too. What use case do you have in mind? I plan to use this for podcast hosting and podcasts are just uploaded once and then downloaded many times. So it's not a problem to block a connection / worker process during upload.
